lib/identity_config.rb

```diff
@@ -358,6 +358,11 @@ def self.build_store(config_map)
     config.add(:usps_ipp_username, type: :string)
     config.add(:usps_ipp_request_timeout, type: :integer)
     config.add(:usps_upload_enabled, type: :boolean)
+    config.add(:get_usps_proofing_results_job_cron, type: :string, allow_nil: true)
+    config.add(
```

Comment:
Personally, I'd be fine if we added a `# rubocop:disable Layout/LineLength` and `# rubocop:enable Layout/LineLength` around all these `config.add` so that we could consistently keep them all one-per-line.

Reply:
It was unintentional to leave in the `allow_nil` option. Once I removed that, Rubocop stopped complaining about the line length.